exporter/internal/factory.go

```diff
@@ -7,31 +7,33 @@ import (
 	"context"
 
 	"go.opentelemetry.io/collector/component"
-) // Factory is a factory interface for exporters.
+)
+
+// Factory is a factory interface for exporters.
 // This interface cannot be directly implemented. Implementations must
 // use the NewFactory to implement it.
 type Factory interface {
 	component.Factory
 
 	// CreateTracesExporter creates a TracesExporter based on this config.
-	// If the exporter type does not support tracing or if the config is not valid,
-	// an error will be returned instead.
+	// If the exporter type does not support tracing,
+	// this function returns the error component.ErrDataTypeIsNotSupported.
```

Comment:
Add a doc link? (same for other mentions of this symbol)

Reply:
sorry, not familiar. What does that look like? My IDE resolves the reference so didn't know I had to do more